vocs.config.ts

```diff
@@ -243,6 +243,10 @@ export default defineConfig({
               },
             ],
           },
+          {
```

Comment:
should we have different items inside this as tutorials?

Reply:
each guide is currently short enough to be part of the same page. Counter MRU tutorial is quite long